plugins/source/test/docs/tables/test_testdata_table.md

```diff
@@ -4,91 +4,78 @@ This table shows data for Test Testdata Table.
\ No newline at end of file
+|float32_list|`list<item: float32, nullable>`|
+|float64_list|`list<item: float64, nullable>`|
+|binary_list|`list<item: binary, nullable>`|
+|string_list|`list<item: utf8, nullable>`|
+|date32_list|`list<item: date32, nullable>`|
+|date64_list|`list<item: date64, nullable>`|
+|duration_s_list|`list<item: duration[s], nullable>`|
+|duration_ms_list|`list<item: duration[ms], nullable>`|
+|duration_us_list|`list<item: duration[us], nullable>`|
+|duration_ns_list|`list<item: duration[ns], nullable>`|
+|interval_month_list|`list<item: month_interval, nullable>`|
+|interval_day_time_list|`list<item: day_time_interval, nullable>`|
+|interval_month_day_nano_list|`list<item: month_day_nano_interval, nullable>`|
+|large_binary_list|`list<item: large_binary, nullable>`|
+|large_string_list|`list<item: large_utf8, nullable>`|
+|time32_s_list|`list<item: time32[s], nullable>`|
+|time32_ms_list|`list<item: time32[ms], nullable>`|
+|time64_us_list|`list<item: time64[us], nullable>`|
+|time64_ns_list|`list<item: time64[ns], nullable>`|
+|timestamp_s_list|`list<item: timestamp[s, tz=UTC], nullable>`|
+|timestamp_ms_list|`list<item: timestamp[ms, tz=UTC], nullable>`|
+|timestamp_us_list|`list<item: timestamp[us, tz=UTC], nullable>`|
+|timestamp_ns_list|`list<item: timestamp[ns, tz=UTC], nullable>`|
+|decimal128_list|`list<item: decimal(19, 10), nullable>`|
+|struct_list|`list<item: struct<int8: int8, int16: int16, int32: int32, int64: int64, uint8: uint8, uint16: uint16, uint32: uint32, uint64: uint64, float32: float32, float64: float64, binary: binary, string: utf8, date32: date32, date64: date64, duration_s: duration[s], duration_ms: duration[ms], duration_us: duration[us], duration_ns: duration[ns], interval_month: month_interval, interval_day_time: day_time_interval, interval_month_day_nano: month_day_nano_interval, large_binary: large_binary, large_string: large_utf8, time32_s: time32[s], time32_ms: time32[ms], time64_us: time64[us], time64_ns: time64[ns], timestamp_s: timestamp[s, tz=UTC], timestamp_ms: timestamp[ms, tz=UTC], timestamp_us: timestamp[us, tz=UTC], timestamp_ns: timestamp[ns, tz=UTC], decimal128: decimal(19, 10)>, nullable>`|
```

Comment:
I wonder why all those changes here?

Reply:
Looks like ordering, with `boolean` type missing for some reason

Reply:
@yevgenypats the changes are because the SDK now doesn't iterate through all Arrow types and adds them, it just adds a static list of predefined types.

Compare https://github.com/cloudquery/plugin-sdk/blob/v3.10.6/schema/testdata.go#L153 vs https://github.com/cloudquery/plugin-sdk/blob/v4.6.1-rc1/schema/testdata.go#L105